packages/react-components/src/components/Picker/Trigger.tsx

```diff
@@ -20,8 +20,9 @@ export interface ITriggerProps {
   isItemSelected: boolean;
   isOpen: boolean;
   isRequired?: boolean;
+  isMultiSelect?: boolean;
   size?: TriggerSize;
-  onClick: () => void;
+  onClick: (e: React.MouseEvent | KeyboardEvent) => void;
```

Comment:
I'm not sure if it's a proper approach to handle both mouse and keyboard events using the `click` name. Either consider `onTrigger` or handle two separate events, as it's misleading.